src/PeopleSDKAdapter.js

```diff
@@ -0,0 +1,120 @@
+  constructor(datasource) {
+    super(datasource);
+
+    this.getPersonObservables = {};
+  }
+
+  /**
+   * Returns a proper Enum key for a given PersonStatus value
+   *
+   * @private
+   * @param {string} status
+   * @returns {string} PersonStatus
+   * @memberof PeopleSDKAdapter
+   */
+  getStatus(status) {
+    return Object.keys(PersonStatus).find((key) => PersonStatus[key] === status);
+  }
+
+  /**
+   * Returns the UUID of a person from the given public ID
+   *
+   * @private
+   * @param {string} ID
+   * @returns {string}
+   * @memberof PeopleSDKAdapter
+   */
+  getPersonUUID(ID) {
+    return deconstructHydraId(ID).id; // ciscospark://us/PEOPLE/<UUID>
+  }
+
+  /**
+   * Fetches the person data from the sdk and returns in the shape required by adapter.
+   *
+   * @private
+   * @param {string} ID
+   * @returns {Person}
+   * @memberof PeopleSDKAdapter
+   */
+  async fetchPerson(ID) {
+    const {id, emails, displayName, firstName, lastName, avatar, orgId} = await this.datasource.people.get(ID);
+
+    return {
+      ID: id,
+      emails,
+      displayName,
+      firstName,
+      lastName,
+      avatar,
+      orgID: orgId,
+    };
+  }
+
+  /**
+   * Returns an observable that emits person data of the given ID.
+   *
+   * @public
+   * @param {string} ID ID of person to get.
+   * @returns {Observable.<Person>}
+   * @memberof PeopleSDKAdapter
+   */
+  getPerson(ID) {
+    if (!(ID in this.getPersonObservables)) {
+      const person$ = from(this.fetchPerson(ID));
+
+      // Subscribe to `Apheleia` internal service to listen for presence changes
+      // And update the person object with status response from the subscription
+      const presenceSubscription$ = from(this.datasource.internal.presence.subscribe(this.getPersonUUID(ID))).pipe(
+        map((data) => data.responses[0].status.status)
+      );
+      const personStatus$ = presenceSubscription$.pipe(
+        flatMap((status) => person$.pipe(map((person) => ({...person, status: this.getStatus(status)}))))
+      );
+
+      // Listen for presence changes for the given person ID after subscription to service
+      const presenceUpdate$ = fromEvent(this.datasource.internal.mercury, USER_PRESENCE_UPDATE_EVENT).pipe(
+        filter((event) => event.data.subject === this.getPersonUUID(ID)),
+        map((event) => this.getStatus(event.data.status))
+      );
+
+      // Update the person status after each change emitted from the event
+      const personUpdate$ = person$.pipe(
+        flatMap((person) => presenceUpdate$.pipe(map((status) => ({...person, status}))))
+      );
+
+      // Fetch the original person data on the first run
+      // and updated object after each status change
+      // include the teardown logic to our finalized observable.
+      // Unsubscribe from `Apheleia` internal service, when
+      // all the subscriptions have been completed.
+      const source = concat(personStatus$, personUpdate$).pipe(
```

Comment:
For the rooms adapter, I went with the naming convention of calling this the name of the function to show it is the final observable:
```suggestion
      const getPerson$ = concat(personStatus$, personUpdate$).pipe(
```